ansible/roles/etcd/tasks/deploy.yml

```diff
@@ -38,24 +38,13 @@
 - name: (re)start etcd
   docker_container:
     name: etcd{{ groups['etcd'].index(inventory_hostname) }}
-    image: quay.io/coreos/etcd:{{ etcd.version }}
+    image: bitnami/etcd:{{ etcd.version }}
     state: started
     recreate: true
     restart_policy: "{{ docker.restart.policy }}"
     volumes: "{{volume_dir | default([])}}"
-    command: "/usr/local/bin/etcd \
```

Comment:
Before merging, I will check how to enable these options with the Bitnami/etcd image.

Reply:
I think this can be addressed in a subsequent PR.